src/eca/handlers.clj

```diff
@@ -49,7 +49,9 @@
                                                                           (config/notify-fields-changed-only!
                                                                            {:chat
                                                                             {:models (sort (keys models))
-                                                                             :behaviors (:chat-behaviors db)
+                                                                             :behaviors (->> (keys (:behavior config))
+                                                                                             (into (:chat-behaviors db))
```

Comment:
Why we need `:chat-behaviors` inside db yet? can't we just drop it and rely 100% on config now?